projects/plugins/jetpack/tools/webpack.config.extensions.js

```diff
@@ -26,19 +26,24 @@ const CopyBlockEditorAssetsPlugin = require( './copy-block-editor-assets' );
  */
 const editorSetup = path.join( path.dirname( __dirname ), 'extensions', 'editor' );
 const viewSetup = path.join( path.dirname( __dirname ), 'extensions', 'view' );
+const blockEditorDirectories = [ 'blocks', 'plugins' ];
 
 /**
- * Filters block scripts
+ * Filters block block scripts
  *
  * @param {string} type - script type
  * @param {string} inputDir - input directory
  * @param {Array} presetBlocks - preset blocks
  * @returns {Array} list of block scripts
  */
-function blockScripts( type, inputDir, presetBlocks ) {
-	return presetBlocks
-		.map( block => path.join( inputDir, 'blocks', block, `${ type }.js` ) )
-		.filter( fs.existsSync );
+function blockEditorScripts( type, inputDir, presetBlocks ) {
+	return _.uniq(
```

Comment:
When would the uniq be needed? Or was it always needed, and was just missing before?

Reply:
It isn't needed/ It was part of the development process and I forgot to remove it. Shouldn't be duplicated paths AFAIK.